lib2/src/component.tsx

```diff
@@ -0,0 +1,5 @@
+import { styled } from 'styled-components';
```

Comment:
import to note the named import of styled rather than defualt import, or

`import {styled}` rather than `import styled`. This is only available in the new beta version of `styled-components`, and is how we can get nextjs to allow importing this module on the server.